packages/base-styles/_mixins.scss

```diff
@@ -114,124 +114,35 @@
-	cursor: default;
-}
-
-@mixin button-style__active() {
-	outline: none;
-	background-color: $white;
-	color: $dark-gray-900;
-	box-shadow: inset 0 0 0 1px $light-gray-700, inset 0 0 0 2px $white;
-}
-
-@mixin button-style__focus-active() {
-	box-shadow: 0 0 0 1px color($theme-color);
-
-	// Windows High Contrast mode will show this outline, but not the box-shadow.
-	outline: 1px solid transparent;
-}
-
-// Switch.
-@mixin switch-style__focus-active() {
-	box-shadow: 0 0 0 2px $white, 0 0 0 3px $dark-gray-300;
+@mixin block-toolbar-button-style__focus() {
+	box-shadow: inset 0 0 0 $border-width-focus $theme-color, inset 0 0 0 4px $white;
+	box-shadow: inset 0 0 0 var(--border-width-focus) $theme-color, inset 0 0 0 4px $white;
 
 	// Windows High Contrast mode will show this outline, but not the box-shadow.
 	outline: 2px solid transparent;
-	outline-offset: 2px;
 }
 
-
-/**
- * Block Toolbar/Formatting Buttons
- */
-
-@mixin block-toolbar-button-style__focus() {
-	box-shadow: inset 0 0 0 2px color($theme-color), inset 0 0 0 4px $white; // Inner halo makes this work on top of a toggled button.
-	outline: 2px solid transparent; // Shown to Windows 10 High Contrast Mode.
-}
-
-
 // Tabs, Inputs, Square buttons.
 @mixin input-style__neutral() {
 	box-shadow: 0 0 0 transparent;
 	transition: box-shadow 0.1s linear;
-	border-radius: $radius-round-rectangle;
-	border: $border-width solid $dark-gray-200;
+	border-radius: $radius-block-ui;
+	border: $border-width solid $medium-gray-text;
 	@include reduce-motion("transition");
 }
 
-@mixin input-style__focus() {
-	color: $dark-gray-900;
-	border-color: $blue-medium-focus;
-	box-shadow: 0 0 0 1px $blue-medium-focus;
-
-	// Windows High Contrast mode will show this outline, but not the box-shadow.
-	outline: 2px solid transparent;
-}
-
-// Square buttons.
-@mixin square-style__neutral() {
-	outline-offset: -1px;
-}
-
-@mixin square-style__focus() {
-	color: $dark-gray-900;
-	outline-offset: -1px;
-	outline: 1px dotted $dark-gray-500;
-}
-
-// Menu items.
-@mixin menu-style__neutral() {
-	border: none;
-	box-shadow: none;
-}
-
-@mixin menu-style__hover() {
-	color: $dark-gray-900;
-	border: none;
-	box-shadow: none;
-	background: $light-gray-200;
-}
-
-@mixin menu-style__focus() {
-	color: $dark-gray-900;
-	border: none;
-	box-shadow: none;
-	outline-offset: -2px;
-	outline: 1px dotted $dark-gray-500;
-}
+$input-focus-width: calc(var(--border-width-focus) - #{ $border-width });
 
-// Blocks in the Library.
-@mixin block-style__disabled {
-	opacity: 0.6;
-	cursor: default;
-}
-
-@mixin block-style__hover {
-	border-color: $theme-color;
-	color: $theme-color !important;
-}
-
-@mixin block-style__focus() {
-	box-shadow: inset 0 0 0 1px $white, 0 0 0 2px $theme-color;
-
-	// Windows High Contrast mode will show this outline, but not the box-shadow.
-	outline: 2px solid transparent;
-}
-
-@mixin block-style__is-active() {
-	color: $white;
-	background: $dark-gray-primary;
+@mixin input-style__focus() {
```

Comment:
These mixins arguably should also be removed. However they are fairly widely used, and in places that are affected by upstream CSS bleed. That's why I omitted these for now. They can be revisited separately.